chain/network/src/routing/routing_table_view/mod.rs

```diff
@@ -111,95 +64,24 @@ impl RoutingTableView {
         self.0.lock().next_hops.len()
     }
 
-    pub(crate) fn find_route(
+    pub(crate) fn find_route_from_peer_id(
```

Comment:
Can you add a comment on what actually is returned? Is it just the first hop on the route?

Reply:
Also sometimes it says find_route(**target**) and sometimes it's find_route_**from**. Can you clarify if those are the same or is it in opposite directions? I get why we need route to a peer but when do we need a route from a peer?

Reply:
I think it is just an unfortunate naming choice; here "from_peer_id" refers to the fact that the argument to the function is a peer_id (in a similar sense to find route "given" peer id).

Will rename to "find_route_to_peer_id" for clarity.

Reply:
I think it is just an unfortunate naming choice; here "from_peer_id" refers to the fact that the argument to the function is a peer_id (in a similar sense to find route "given" peer id).

I've renamed it to `find_next_hop_for_target`.